stencil.config.ts

```diff
@@ -19,7 +20,7 @@ export const config: Config = {
       type: 'dist',
     },
     {
-      type: 'dist-custom-elements-bundle',
+      type: 'dist-custom-elements',
```

Comment:
Can we so "easily" remove the "-bundle"? https://github.com/lyne-design-system/lyne-components/issues/946

Reply:
I'm actually not sure.
@DavideMininni-Fincons If I remember correctly, quite a while ago you checked what the differences were. Do you remember any details?

Reply:
The issue about deprecating dist-custom-elements-bundle is still open. Link: https://github.com/ionic-team/stencil/issues/3136
It seems that there are still some bugs that Stencil team should fix.

Reply:
I'll revert it for now.